app/code/Magento/Customer/ViewModel/Address.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Customer\ViewModel;
+
+use Magento\Directory\Helper\Data as DataHelper;
+use Magento\Customer\Helper\Address as AddressHelper;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+
+/**
+ * Customer address view model.
+ */
+
+ /**
+  * Address view model
+  */
+class Address implements ArgumentInterface
+{
+    /**
+     * Data helper
+     *
+     * @var DataHelper
+     */
+    private $helperData;
+
+    /**
+     * Address helper
+     *
+     * @var AddressHelper
+     */
+    private $helperAddress;
+
+    /**
+     * Constructor
+     *
+     * @param DataHelper $helperData
+     * @param AddressHelper $helperAddress
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function __construct(
+        DataHelper $helperData,
+        AddressHelper $helperAddress
+    ) {
+        $this->helperData= $helperData;
+        $this->helperAddress= $helperAddress;
+    }
+
+    /**
+     * Get string with frontend validation classes for attribute
+     *
+     * @param string $attributeCode
+     *
+     * @return string
```

Comment:
remove blank line

Reply:
Afaik the linting / PHPCS had a problem with the missing empty line. Check the failing checks from the previous CI builds.